extensions/ARB/ARB_gl_spirv.txt

```diff
@@ -983,6 +984,17 @@ Add Appendix A.spv "The OpenGL SPIR-V Execution Environment" to the OpenGL 4.5
       - Workgroup
       - Invocation
 
+    *Storage Class* must be limited to:
+      - *UniformConstant*
+      - *Input*
+      - *Uniform*
+      - *Output*
+      - *Workgroup*
+      - *Private*
+      - *Function*
+      - *PushConstant*
```

Comment:
Did OpenGL add support for push_constant?

Reply:
Oops. Replaced PushConstant with AtomicStorage